dotnet/test/ColumnStoreDriverTest.cs

```diff
@@ -0,0 +1,294 @@
+            {
+                throw new ArgumentException(
+                        "The path specified in the environment variable "
+                        + ColumnStoreInstallEnvVariable
+                        + " ("
+                        + columnstoreInstallDir
+                        + ") does not exist!"
+                        + " (Please be sure to specify an absolute path.)"
+                    );
+            }
+            else
+            {
+                ColumnstoreXmlFilePath = Path.Combine(columnstoreInstallDir, "etc", ColumnStoreXmlFilename);
+            }
+
+            if (string.IsNullOrWhiteSpace(ColumnstoreXmlFilePath))
+            {
+                throw new ArgumentException(
+                        "Did not find "
+                        + ColumnStoreXmlFilename
+                        + " Please set the environment variable "
+                        + ColumnStoreInstallEnvVariable
+                        + " to a directory containing this file."
+                    );
+            }
+            else if (!File.Exists(ColumnstoreXmlFilePath))
+            {
+                throw new ArgumentException(
+                        ColumnstoreXmlFilePath
+                        + " does not exist? Please check/set the environment variable "
+                        + ColumnStoreInstallEnvVariable
+                    );
+            }
+
+            if (IsLinux)
+            {
+
+            }
+            else
+            {
+                // Windows
+                if (!PathContainsWindowsDlls())
+                {
+                    throw new ArgumentException(
+                        "Did not find all of the needed DLLs in your PATH."
+                        + " SOURCE_DIR/dotnet/lib-win64 needs to be in the PATH"
+                        + " and contain:"
+                        + " dotnet_mcsapi.dll mcsapi.dll libiconv.dll libuv.dll libxml2.dll"
+                    );
+                }
+            }
+        }
+
+        // public static string GetExecutingDirectoryName()
+        // {
+        //     var location = new Uri(Assembly.GetEntryAssembly().GetName().CodeBase);
+        //     return new FileInfo(location.AbsolutePath).Directory.FullName;
+        // }
+
+
+        /// <summary>
+        /// Crude way to check for needed DLLs. For windows, the DLLs need to be
+        /// in the PATH. They cannot be in the same directory as the .NET DLLs as
+        /// the c++ DLLs are unmanaged and putting them in the .NET assembly directory
+        /// will produce hard-to-diagnose errors.
+        /// </summary>
+        public static bool PathContainsWindowsDlls()
+        {
+            var currentPaths = Environment.GetEnvironmentVariable("PATH").Split(Path.PathSeparator);
+            for (int i = 0; i < currentPaths.Length; i++)
+            {
+                if (currentPaths[i].EndsWith(@"\dotnet\lib-win64") || currentPaths[i].EndsWith("/dotnet/lib-win64"))

```

Comment:
change to "\dotnet\lib-win-x64" and "/dotnet/lib/win-x64" to be consistent with documentation